libsolidity/interface/StandardCompiler.cpp

```diff
@@ -996,13 +996,13 @@ Json::Value StandardCompiler::compile(Json::Value const& _input) noexcept
 	}
 }
 
-string StandardCompiler::compile(string const& _input) noexcept
+string StandardCompiler::compile(string _input) noexcept
 {
 	Json::Value input;
 	string errors;
 	try
 	{
-		if (!jsonParseStrict(_input, input, &errors))
+		if (!jsonParseStrict(std::move(_input), input, &errors))
```

Comment:
I think all versions of  `jsonParseStrict` take a `const&` so there is no point in removing the `const&` from `_input`.